deepchecks/checks/methodology/boosting_overfit.py

```diff
@@ -118,7 +118,7 @@ def calculate_steps(num_steps, num_estimators):
 
 
 class BoostingOverfit(TrainTestBaseCheck):
-    """Check for overfit occurring when increasing the number of iterations in boosting models.
+    """Check for overfit caused by using too many boosting iterations in a gradient boosted model.
```

Comment:
"boosting iterations" is a bit weird, especially when you later affirm that you're talking about gradient boosting.
WDYT?

Reply:
mmm not sure I understand why that is weird. What is the alternative you suggest?

Reply:
Dropping the first use of "boosting". Just saying "too many iterations"